WooCommerce/Classes/ViewRelated/Orders/Order Details/Issue Refunds/IssueRefundViewModel.swift

```diff
@@ -75,7 +75,12 @@ final class IssueRefundViewModel {
     /// Creates the `ViewModel` to be used when navigating to the page where the user can
     /// confirm and submit the refund.
     func createRefundConfirmationViewModel() -> RefundConfirmationViewModel {
-        RefundConfirmationViewModel(order: state.order, currencySettings: state.currencySettings)
+        let refundItems = state.refundQuantityStore.map { RefundableOrderItem(item: $0, quantity: $1) }
```

Comment:
nit: not sure if it's worth creating a variable in `RefundQuantityStore`'s extension for this mapping, since it's used a few times?